Examples/Physics_applications/laser_acceleration/PICMI_inputs_1d.py

```diff
@@ -51,6 +51,31 @@
     name = 'electrons',
     initial_distribution = uniform_distribution)
 
+# Particles: beam electrons
+q_tot = 1e-12
+x_m = 0.
+y_m = 0.
+z_m = -28e-06
+x_rms = 0.5e-06
+y_rms = 0.5e-06
+z_rms = 0.5e-06
+ux_m = 0.
+uy_m = 0.
+uz_m = 500.
+ux_th = 2.
+uy_th = 2.
+uz_th = 50.
+gaussian_bunch_distribution = picmi.GaussianBunchDistribution(
+    n_physical_particles = q_tot / q_e,
+    rms_bunch_size = [x_rms, y_rms, z_rms],
+    rms_velocity = [c*ux_th, c*uy_th, c*uz_th],
+    centroid_position = [x_m, y_m, z_m],
+    centroid_velocity = [c*ux_m, c*uy_m, c*uz_m])
+beam = picmi.Species(
+    particle_type = 'electron',
+    name = 'beam',
+    initial_distribution = gaussian_bunch_distribution)
+
```

Comment:
@prkkumar Do you happen to see anything wrong with this beam setup in 1D? I checked the Gaussian beam implementation in 1D and it seems fine to me, however this beam setup seems to produce checksum values for the beam particle attributes that are all zero (see file Regression/Checksum/benchmarks_json/Python_LaserAcceleration_1d.json below). In 2D and 3D the same checksum values are non-zero instead, as expected. Will also check the 1D code for `GaussianBunchDistribution` on the PICMI side in the meantime.

Reply:
I do not see anything wrong with this setup. Not sure if picmi.GaussianBunchDistribution function needs anything special for 1D though.

Reply:
Thanks. I don't see anything specific to 1D (nor any other dimensionality) in `GaussianBunchDistribution` on the PICMI side. I also printed out some of the position and momentum values from within `PhysicalParticleContainer::AddGaussianBeam` (for the beam injection) and they seemed fine to me. I haven't yet understood why the beam checksum values are all zero.